app/src/main/java/com/spoony/spoony/core/designsystem/component/snackbar/SpoonySnackbar.kt

```diff
@@ -0,0 +1,105 @@
+package com.spoony.spoony.core.designsystem.component.snackbar
+
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.Button
+import androidx.compose.material3.Scaffold
+import androidx.compose.material3.SnackbarHost
+import androidx.compose.material3.SnackbarHostState
+import androidx.compose.material3.Surface
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.CompositionLocalProvider
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.rememberCoroutineScope
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.spoony.spoony.core.designsystem.event.LocalSnackBarTrigger
+import com.spoony.spoony.core.designsystem.theme.SpoonyAndroidTheme
+import kotlinx.coroutines.delay
+import kotlinx.coroutines.launch
+
+@Composable
+fun SpoonySnackbar(
+    text: String,
+    modifier: Modifier = Modifier
+) {
+    Surface(
+        modifier = modifier
+            .padding(bottom = 20.dp, start = 20.dp, end = 20.dp),
```

Comment:
```suggestion
            .padding(
                bottom = 20.dp,
                start = 20.dp,
                end = 20.dp
            ),
```
P4: 이런식으로 줄바꿈 해주는 것이 더 가독성이 좋을 것 같아요!

Reply:
P1: 패딩 값 피그마랑 다른 것 같은데 다시 한번만 확인 부탁드릴게요..!!

Reply:
협의 완료